src/platform/silabs/SiWx917/siwx917-chip-mbedtls-config.h

```diff
@@ -115,38 +91,21 @@ typedef void mbedtls_ecp_restart_ctx;
 #define MBEDTLS_SSL_EXPORT_KEYS
 #define MBEDTLS_SSL_KEEP_PEER_CERTIFICATE
 #define MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
-#define MBEDTLS_SSL_PROTO_TLS1_2
 #define MBEDTLS_SSL_PROTO_DTLS
+#define MBEDTLS_SSL_PROTO_TLS1_2
 #define MBEDTLS_SSL_SRV_C
 #define MBEDTLS_SSL_TLS_C
-#define MBEDTLS_X509_CREATE_C
-#define MBEDTLS_X509_CSR_WRITE_C
-#define MBEDTLS_X509_CRL_PARSE_C
-#define MBEDTLS_X509_CRT_PARSE_C
-#define MBEDTLS_X509_CSR_PARSE_C
-#define MBEDTLS_X509_USE_C
-
-#define MBEDTLS_MPI_WINDOW_SIZE 1 /**< Maximum windows size used. */
-#ifdef DIC_ENABLE
-#define MBEDTLS_MPI_MAX_SIZE 512 /**< Maximum number of bytes for usable MPIs. */
-#else
-#define MBEDTLS_MPI_MAX_SIZE 32 /**< Maximum number of bytes for usable MPIs. */
-#endif
-#define MBEDTLS_ECP_MAX_BITS 256        /**< Maximum bit size of groups */
-#define MBEDTLS_ECP_WINDOW_SIZE 2       /**< Maximum window size used */
-#define MBEDTLS_ECP_FIXED_POINT_OPTIM 0 /**< Enable fixed-point speed-up */
-#define MBEDTLS_ENTROPY_MAX_SOURCES 2   /**< Maximum number of sources supported */
-
-#if OPENTHREAD_CONFIG_COAP_SECURE_API_ENABLE
-#define MBEDTLS_SSL_MAX_CONTENT_LEN 900 /**< Maxium fragment length in bytes */
-#else
-#ifdef DIC_ENABLE
-#define MBEDTLS_SSL_MAX_CONTENT_LEN 5120 /**< Maxium fragment length in bytes */
-#else
-#define MBEDTLS_SSL_MAX_CONTENT_LEN 768 /**< Maxium fragment length in bytes */
-#endif                                  // DIC
+#define MBEDTLS_TRNG_C
+
+#if defined(MBEDTLS_ECP_ALT) && !defined(MBEDTLS_ECP_RESTARTABLE)
+typedef void mbedtls_ecp_restart_ctx;
 #endif
 
-#define MBEDTLS_SSL_CIPHERSUITES MBEDTLS_TLS_ECJPAKE_WITH_AES_128_CCM_8
+#endif // SL_MBEDTLS_USE_TINYCRYPT
+
+// SLC GENERATED
+#include "sl_mbedtls_config.h"
 
+#if SL_MBEDTLS_USE_TINYCRYPT
 #include "mbedtls/check_config.h"
+#endif // SL_MBEDTLS_USE_TINYCRYPT
```

Comment:
I don't think we need this. the `sl_mbedtls_config.h` is responsible to include this.